tests/org.eclipse.n4js.json.testsuite/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016 NumberFour AG.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+  NumberFour AG - Initial API and implementation
+-->
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>org.eclipse.n4js.json.testsuite</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<parent>
+		<groupId>org.eclipse.n4js</groupId>
+		<artifactId>org.eclipse.n4js.tests</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<build>
+		<plugins>
+		</plugins>
+	</build>
+	
+	<profiles>
+		<profile>
+			<!-- JSONTestSuite -->
+			<id>execute-json-testsuite</id>
+			<activation><activeByDefault>true</activeByDefault></activation>
```

Comment:
Note: "activeByDefault" activation might be misleading here. 
Basically as we use other profiles a lot, this profile will *effectively deactivated*. 

C.f.: https://maven.apache.org/guides/introduction/introduction-to-profiles.html  and the section about `activeByDefault`
This profile will automatically be active for all builds *unless another profile in the same POM is activated using one of the previously described methods*. All profiles that are active by default are automatically deactivated when a profile in the POM is activated on the command line or through its activation config.